proposed/0021-MPID-Messaging-delete/0021-MPID-Messaging-delete.md

```diff
@@ -0,0 +1,75 @@
+- Feature Name: Deletion
```

Comment:
This is perhaps not descriptive enough, the RFC is for a particular purpose I think.